src/dropdown.option.test.interactions.single.ts

```diff
@@ -95,54 +95,6 @@ it('has no tooltip when active and with a short label set programmatically', asy
   expect(tooltip?.open).to.be.false;
 });
 
-it('sets `aria-selected` when selected programmatically', async () => {
```

Comment:
Because Dropdown Option now takes into account whether it's the last selected option when setting `aria-selected`, I had to move these tests to [`dropdown.test.interactions.single.ts`](https://github.com/CrowdStrike/glide-core/pull/929/files) and [`dropdown.test.interactions.multiple.ts`](https://github.com/CrowdStrike/glide-core/pull/929/files#diff-83a2f8e5f509ee527d6f3b723130a670bda9219cceddf361e279cf10a8b1a4d0R1567-R1696). Because it didn't make sense for an isolated Dropdown Option test suite to test multiple Dropdown Options in a Dropdown.